### ISSUE TITLE
Navigate by sentence during say all

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2880]
**Reported by amenaide on 2012-12-21 13:52**
I would like to be able to skip back and forth between sentences, paragraphs etc during read all. This would be helpful when reading long texts quickly where not all information is important to the reader.

**Comment by bhavyashah** [comment]
If I am following this ticket correctly, the 'Allow skim reading during say all' option in the Keyboard Settings dialog does exactly this. It allows for standard navigation during a say all and adjusting the flow of the say all accordingly. When it comes to sentence navigation, this has been discussed elsewhere. bhirmanpour's request to set a dedicated speech rate for say alls may depend on the speech refactor that is being worked on for NVDA, and ideally, should be covered by a separate ticket. All in all, a portion of this ticket is already achievable, a portion is blocked by another ticket's implementation, and a portion duplicates another ticket. Thus, I suggest closing (and providing the numbers for related tickets for reference purposes). @ehollig

**Comment by ehollig** [comment]
The Allow skim reading during say all' option will allow you to move up and down by paragraphs during say all. I just tested this functionality on this page after indicating a say all.
Dedicated speech rate for say all can be achieved using profiles.
For sentence movement in browse mode, this is #5463.
For other applications or supporting the moving by sentence natively by NVDA, this is covered in #1873.
Closing
